dataland-frontend/tests/e2e/utils/CompanyUpload.ts

```diff
@@ -88,3 +88,15 @@ export async function uploadCompanyViaApi(
   );
   return data.data;
 }
+
+/**
+ * A simple util to create a unique company
+ * @param companyName the company name which will be suffixed with a unique ID
+ * @returns company information with unique identifiers
+ */
+export function prepareUniqueCompany(companyName: string): CompanyInformation {
+  const uuid = new Date().getTime();
+  const uniqueCompanyName = `${companyName}-${uuid}`;
+
```

Comment:
After adjustments made by my other review comments, I think you won't need this function anymore.

Reply:
Code duplication. Used the same code in multiple tests. So I created a util function.

Reply:
Originally you were right.

But after we switch from company-name assertion to URL-assertion in the function `testEditDataButton()` in the test file `EditButtonWithMultipleReportingPeriods.ts`, there is only one occurrence left of `prepareUniqueCompany()`, which is in the `CheckQaProcedure.ts` test file.

So then it is no code duplication anymore and we can remove it for now.

However, it is not good that in `main` in `CheckQaProcedure.ts`, we write:
`const uuid = new Date().getTime()`
I think that variable should be renamed to `timestamp` or smth similar, because the word `uuid` indicates that it is an actual UUID format: `18175bc0-66e2-4be9-9a73-4b8961eda451`

Reply:
`prepareUniqueCompany()` is being used in two files... `EditButtonWithMultipleReportingPeriods.ts` and `CheckQaProcedure.ts`

I can remove it, but believe it may be a useful function for creating unique company names.

Reply:
Although... I've just reduced it to a one-liner.. I'll remove it.

Reply:
👍